ui/app/components/ui/page-container/index.scss

```diff
@@ -91,24 +92,24 @@
   }
 
   &__back-button {
+    @include Paragraph;
+
     color: #2f9ae0;
-    font-size: 1rem;
     cursor: pointer;
-    font-weight: 400;
   }
 
   &__title {
+    @include H2;
+
     color: $black;
-    font-size: 2rem;
     font-weight: 500;
-    line-height: 2rem;
```

Comment:
the resulting line height is much higher, as noted by @Gudahtt in #9074, but on review:

> The `line-height` is now much higher with this change (it went from 32px to 44.8px), but I do think this looks better in all cases I know of. I can't find any with stray padding that is now redundant.


https://github.com/MetaMask/metamask-extension/pull/9074/files?file-filters%5B%5D=.scss#r466411684